saleor/graphql/account/types.py

```diff
@@ -252,6 +256,13 @@ def resolve_avatar(root: models.User, info, size=None, **_kwargs):
                 info=info,
             )
 
+    @staticmethod
+    @login_required
+    def resolve_stored_payment_sources(root: models.User, _):
```

Comment:
By convention, we use `_info`

```suggestion
    def resolve_stored_payment_sources(root: models.User, _info):
```